gcp_deepvariant_runner_test.py

```diff
@@ -520,6 +520,668 @@ def testRunFailsCannotWriteOutputBucket(self, mock_blob_exists):
     with self.assertRaises(ValueError):
       gcp_deepvariant_runner.run(self._argv)
 
+  def testRunFailsSetOptimizedFlagsMissingExpectedModel(self):
+    self._argv.extend(['--set_optimized_flags_based_on_bam_size'])
+    with self.assertRaises(ValueError):
+      gcp_deepvariant_runner.run(self._argv)
+
+  def testRunFailsSetOptimizedFlagsMissingBamFile(self):
+    self._argv = [
```

Comment:
I would very much like to see this duplication removed and replaced with a utility function that generates them and then we will only see the parts that are expected to differ for each test

Reply:
done.